### ISSUE TITLE
Generator syntax in block finding

### ISSUE THREAD

**Issue by MikeInnes** [JunoLab/Atom.jl#108]
e.g.
```julia
module Foo

all(p isa TrackedArray for p in params(cm))

end
```
tries to grab the last `end` as part of the block.

**Comment by pfitzseb** [comment]
Are you on CodeTools.jl matter? I thought I'd fixed that already...

**Comment by pfitzseb** [comment]
Fixed on julia#0.7 branch of julia-client.
